### PR TITLE
feat: add expo monorepo example

### LINKED ISSUE #172: Monorepo support via yarn workspaces
**Describe the example you'd like**

It'd be good to provide an example with a similar layout to `expo/expo` and leveraging expo-yarn-workspaces.

**Any existing examples?**

https://github.com/Fried-Chicken/expo-monorepo - out of date

### PR DESCRIPTION
**Pull request by nomi9995**
Fixes #172 

Hi, here's a small PR for add Expo Yarn Workspaces. Let me know if have to add detailed readme or anything in with-monorepo.

### CODE REVIEW

**Comment by nomi9995** [comment]
@brentvatne will you please review my PR

**Comment by brentvatne** [comment]
hi! sorry i will check this out soon

**Comment by brentvatne** [comment]
i think this could benefit from a couple changes:

- delete example2, include a single example in a `/apps` directory
- create a library in the `/packages` directory and use it in the app

File: with-monorepo/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "scripts": {
+    "postinstall": "./postinstall.js"
```
Comment (brentvatne) [inline with-monorepo/package.json:3]: i would recommend this instead: https://github.com/expo/expo/blob/master/package.json#L10

**Pull request closed without merging**